pyproject.toml

```diff
@@ -21,7 +21,7 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "pydantic>=1.0"
+    "pydantic>=1.7,<2.0"
```

Comment:
Are we sure 2.0 will be breaking for our use case? I'm hesitant to add the upper version restriction.